src/quadrature/ftest/tstquadrature_interfaces.f90

```diff
@@ -1,47 +1,46 @@
-!----------------------------------*-F90-*----------------------------------
+!---------------------------------------------*-F90-*-----------------------------------------------
 !
 ! file   quadrature/ftest/tstquadrature_interfaces.f90
 ! author Allan Wollaber
 ! date   Tuesday, Jun 12, 2012, 16:03 pm
 ! brief  Test F90 quadrature_data passed into a C++ function
-! note   Copyright (c) 2016-2020 Triad National Security, LLC.
-!        All rights reserved.
-!---------------------------------------------------------------------------
-
-!---------------------------------------------------------------------------
-! This F90 function interface allows the Fortran function
-! test_quadrature_interfaces to directly call the extern "C" function
-! rtt_test_quadrature_interfaces (from the library rtt_quadrature_test).
-!---------------------------------------------------------------------------
+! note   Copyright (c) 2016-2021 Triad National Security, LLC., All rights reserved.
+!---------------------------------------------------------------------------------------------------
+
+!---------------------------------------------------------------------------------------------------
+! This F90 function interface allows the Fortran function test_quadrature_interfaces to directly
+! call the extern "C" function rtt_test_quadrature_interfaces (from the library
+! rtt_quadrature_test).
+!---------------------------------------------------------------------------------------------------
 module rtt_test_quadrature_f
   implicit none
 
-  ! Now create an interface to the C routine that accepts that a user-defined
-  ! type
+  ! Now create an interface to the C routine that accepts that a user-defined type
   interface
-     subroutine rtt_test_quadrature_interfaces(quad, err_code) bind(C, &
```

Comment:
Intel/20 didn't like this one line.  The rest of the changes are from a new F90 code formatting tool that I'm playing with.